css/login.css

```diff
@@ -0,0 +1,29 @@
+ody {
+    box-sizing: border-box;
+}
+
+#external {
+    background-image:url(../assets/feedback-banner.jpg);
+        
+}
+
+img {
+/*    float:left;*/
```

Comment:
Do you still need this rule here? If not, it's a best practice to clean up after yourself when you're done. Keeps the code nice and tidy and improves overall legibility.